modules/archetypes/lib/policy_definitions/policy_definition_es_deploy_asc_standard.json

```diff
@@ -211,7 +271,7 @@
                     "apiVersion": "2018-06-01",
                     "name": "StorageAccounts",
                     "dependsOn": [
-                      "[concat('Microsoft.Security/pricings/VirtualMachines')]"
+                      "[[concat('Microsoft.Security/pricings/VirtualMachines')]"
```

Comment:
Error from parsing in template creation process. Needs fix to `ToTemplateFile()` method in `ArmTemplateResource` class.